src/user/models/user/user.service.ts

```diff
@@ -132,6 +132,8 @@ export class UserService {
   private async checkIpCountry(userIp: string): Promise<string> {
     const ipCountry = await this.geoLocationService.getCountry(userIp);
 
+    console.log(`SignUp User-Ip ${userIp}; Ip-country ${ipCountry}`);
+
```

Comment:
I added this log in a different branch just for testing purposes. It can be removed here.